src/virtual_maize_field/world_generator/field.world.template

```diff
@@ -165,795 +163,39 @@
-      <visual name="visual_code">
-	<pose>0 0 0 0 0 0</pose>
-        <geometry>
-          <box>
-            <size>1 1 1</size>
-          </box>
-        </geometry>
-        <material>
-          <script>
-            <uri>model://location_marker_a/materials/scripts</uri>
-            <uri>model://location_marker_a/materials/textures</uri>
-            <name>location_marker_a/qr_code</name>
-          </script>
-        </material>
-      </visual>
-    </link>
-  </model>
-
-{% elif coordinate.type == 'location_marker_b' or coordinate.type == 'ghost_location_marker_b' %}
-<!--location_marker_b-->
-    <model name='{{ coordinate.name }}'>
-      <pose frame=''>{{ coordinate.x }} {{ coordinate.y }} {{ coordinate.z }} 0 0 0</pose>
-      <static>true</static>
-      <link name='{{ coordinate.name }}_link'>
-      <pose>0 0 1 0 0 0</pose>
-      <collision name="collision">
-        <geometry>
-          <box>
-            <size>1 1 2</size>
-          </box>
-        </geometry>
-      </collision>
-      <visual name="foot">
-	<pose>0 0 -1.5 0 0 0</pose>
-        <geometry>
-          <box>
-            <size>1 1 1</size>
-          </box>
-        </geometry>
-      </visual>
-      <visual name="visual_bottom">
-	<pose>0 0 -0.75 0 0 0</pose>
-        <geometry>
-          <box>
-            <size>1 1 0.5</size>
-          </box>
-        </geometry>
-        <material>
-          <script>
-            <uri>model://location_marker_b/materials/scripts</uri>
-            <uri>model://location_marker_b/materials/textures</uri>
-            <name>location_marker_b/logo</name>
-          </script>
-        </material>
-      </visual>
-      <visual name="visual_top">
-	<pose>0 0 0.75 0 0 0</pose>
-        <geometry>
-          <box>
-            <size>1 1 0.5</size>
-          </box>
-        </geometry>
-        <material>
-          <script>
-            <uri>model://location_marker_b/materials/scripts</uri>
-            <uri>model://location_marker_b/materials/textures</uri>
-            <name>location_marker_b/logo</name>
-          </script>
-        </material>
-      </visual>
-      <visual name="visual_code">
-	<pose>0 0 0 0 0 0</pose>
-        <geometry>
-          <box>
-            <size>1 1 1</size>
-          </box>
-        </geometry>
-        <material>
-          <script>
-            <uri>model://location_marker_b/materials/scripts</uri>
-            <uri>model://location_marker_b/materials/textures</uri>
-            <name>location_marker_b/qr_code</name>
-          </script>
-        </material>
-      </visual>
-    </link>
-  </model>
-
-{% endif %}
-{% endfor %}
-
+    {# All other models can be included by using the SDF #}
+    {% else %}
+    <include>
+      <uri>model://{{ coordinate.type }}</uri>
+      <pose>{{ coordinate.x }} {{ coordinate.y }} {{ coordinate.z }} {{ coordinate.roll }} {{ coordinate.pitch }} {{ coordinate.yaw }}</pose>
+      {% if coordinate.type == "maize_01" or coordinate.type == "maize_02" -%}
```

Comment:
@Darkproduct, I though this part would rescale the maize plants again, but apparently thats not working

Reply:
That seems to be not possible. Best approach is to rescale the maize plants in the SDF I think.